CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- Install Banner PWA.
```

Comment:
Shouldn't this be only visible on mobile? Also, shouldn't this be at the top? cc: @alinevillaca

Reply:
Maybe this should be a separated app that can be used through out the store. Think about it...

Reply:
@salesfelipe Maybe I'll rewrite this to be a mini info bar at the top. What do you think?

Reply:
The location should be defined by the designers. Talk to @alinevillaca and see what she thinks...